Resources/config/services.xml

```diff
@@ -185,10 +184,10 @@
             <argument type="constant">NULL</argument> <!-- expression evaluator -->
 
             <call method="setSerializationContextFactory">
-                <argument type="service" id="jms_serializer.serialization_context_factory" />
+                <argument type="service" id="jms_serializer.context_factory" />
```

Comment:
you will need it since you have serialization context and deserialization context (that might have different configurations), you will need two different instances